lib/runners/config_generator.rb

```diff
@@ -8,7 +8,7 @@ def generate(tools: [])
     private
 
     def load_template(tools)
-      filename = __FILE__.sub(".rb", ".yml.erb")
+      filename = __FILE__.gsub(/.rb\Z/, ".yml.erb")
```

Comment:
[nitpick] Escaping `.` seems more accurate, but this is enough 👍 

```suggestion
      filename = __FILE__.gsub(/\.rb\Z/, ".yml.erb")
```